utils/check_doc_readme.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+ROOT_DIR=$1
+DOC_DIR=$1/docs
+
+README="README.md"
+
+echo "Checking whether docs/README.md is up-to-date or not ..."
+
+diff ${ROOT_DIR}/${README} ${DOC_DIR}/${REAMDE} > readme.patch
+
```

Comment:
Just curious how about using the following script without the temporary file readme.patch:
```
diff ${ROOT_DIR}/${README} ${DOC_DIR}/${REAMDE}
if [ $? == 0 ] 
then
...
else
...
fi

Reply:
It looks working well. Thank you for the suggestion! I did not know about this.